testcases/kernel/syscalls/fdatasync/fdatasync03.c

```diff
@@ -45,12 +42,10 @@ static void verify_fdatasync(void)
 	if (TST_RET)
 		tst_brk(TFAIL | TTERRNO, "fdatasync(fd) failed");
 
-       written = tst_dev_bytes_written(dev);
+	written = tst_dev_bytes_written(dev);
```

Comment:
indentation

Reply:
Actually indentation was wrong before, now it is correct :)

Reply:
@shaikshavali1 I assume LTP_DEV will have the /dev/vdb in it. But how about tst_dev_bytes_written, what should we pass to it?